.travis.yml

```diff
@@ -34,29 +33,29 @@ matrix:
     # Absolute minimum dependencies
     - python: 2.7
       env:
-        - DEPENDS="numpy==1.7.1" PYDICOM=0
+        - DEPENDS="numpy==1.7.1"
```

Comment:
Wasn't the idea here to make sure that we didn't break when pydicom was absent? Do we want to move to a pydicom dependency?

Reply:
Wow, brain failure. I see this replaces the pydicom dependency above. Carry on.

Reply:
With this matrix entry, Pydicom isn't in the DEPENDS variable, and doesn't get installed.  On the way, the tests did in fact fail when Pydicom wasn't present, I had to fix the code to make it work again.  Have a look at the Travis-CI log to confirm this entry does not install Pydicom...